docs/showcase.md

```diff
@@ -26,5 +26,5 @@ Want to see what Quartz can do? Here are some cool community gardens:
 - [Data Dictionary 🧠](https://glossary.airbyte.com/)
 - [sspaeti.com's Second Brain](https://brain.sspaeti.com/)
 - [🪴Aster's notebook](https://notes.asterhu.com)
-
+- [Saurabh's Digital Garden](https://www.saurabhsinghrajput.com/)
```

Comment:
Can you add a single empty line after your link? This way the GitHub Action pipelines can succeed.